app/controllers/api/database.php

```diff
@@ -206,8 +206,8 @@ function createAttribute($collectionId, $attribute, $response, $dbForInternal, $
     ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
     ->label('sdk.response.model', Response::MODEL_COLLECTION_LIST)
     ->param('search', '', new Text(256), 'Search term to filter your list results. Max length: 256 chars.', true)
-    ->param('limit', 25, new Range(0, 100), 'Results limit value. By default will return maximum 25 results. Maximum of 100 results allowed per request.', true)
-    ->param('offset', 0, new Range(0, APP_LIMIT_COUNT), 'Results offset. The default value is 0. Use this param to manage pagination.', true)
+    ->param('limit', 25, new Range(0, 100), 'Maximum number of collection to return in response. Use this value to manage pagination. By default will return maximum 25 results. Maximum of 100 results allowed per request.', true)
```

Comment:
Might be a good idea to use the pagination related params description for linking to our new upcoming pagination section in the docs

Reply:
Adapted these 👍🏻